agent/calculation/simple_area.py

```diff
@@ -5,8 +5,10 @@
 from twa import agentlogging
 from tqdm import tqdm
 import sys
+from agent.objects.exposure_value import ExposureValue
 
 logger = agentlogging.get_logger('dev')
+unit = 'm²'
```

Comment:
You seem to be defining this in more than one place. Please consolidate this into a single definition that is being reused.